implementations/py/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@polywrap/python-fs-plugin",
   "description": "Polywrap Python Fs Plugin",
-  "version": "0.10.1",
+  "version": "0.10.5",
```

Comment:
Just FYI, for `package.json`'s that are only used for the purpose of installing the `polywrap` CLI, we don't need to add a version since it will never be published. Here's an example: https://github.com/polywrap/wrap-abi-bindgen/blob/dev/implementations/app-typescript/package.json